src/components/CommandPalette.tsx

```diff
@@ -421,18 +447,26 @@ const CommandPalette: FC = () => {
               ...(!isTouch ? { maxHeight: 'calc(100vh - 8em)', overflow: 'auto' } : null),
             })}
           >
-            {shortcuts.map(shortcut => (
-              <CommandRow
-                search={search}
-                gestureInProgress={gestureInProgress as string}
-                key={shortcut.id}
-                last={shortcut === shortcuts[shortcuts.length - 1]}
-                onClick={onExecute}
-                onHover={onHover}
-                selected={!isTouch ? shortcut === selectedShortcut : gestureInProgress === shortcut.gesture}
-                shortcut={shortcut}
-              />
-            ))}
+            {shortcuts.map(shortcut => {
+              // Check if the current gesture sequence ends with help gesture
+              const isHelpMatch =
+                shortcut.id === 'help' && (gestureInProgress as string)?.toString().endsWith(helpGesture)
```

Comment:
Let's use [gestureString](https://github.com/cybersemics/em/blob/75b8c516e70922bdfa44d29a8cc47e33c942e865/src/commands.ts#L321-L323) and [commandById](https://github.com/cybersemics/em/blob/75b8c516e70922bdfa44d29a8cc47e33c942e865/src/commands.ts#L325-L326) here as well.